errors/http.go

```diff
@@ -0,0 +1,127 @@
+// Package backoff is a slightly changed version of the backoff algorithm that comes with gRPC
+// See https://github.com/grpc/grpc-go/blob/master/LICENSE for the license
+
+package errors
+
+import (
+	"encoding/json"
+	"net/http"
+)
+
+// CodeHeader is the header where the error code will be stored
+const CodeHeader = "X-TTN-Error-Code"
+
+// HTTPStatusCode returns the corresponding http status code from an error type
+func (t Type) HTTPStatusCode() int {
```

Comment:
Let's keep this in sync with https://github.com/grpc-ecosystem/grpc-gateway/blob/7195ea445241f9773e71fc7b94c306d3ed6b35b6/runtime/errors.go#L14-L55